OpenRA.Mods.Common/Traits/GivesExperience.cs

```diff
@@ -49,6 +52,11 @@ public void Killed(Actor self, AttackInfo e)
 			var killer = e.Attacker.TraitOrDefault<GainsExperience>();
 			if (killer != null)
 				killer.GiveExperience(exp);
+
+			if (info.GivesPlayerExperience)
+			{
+				e.Attacker.Owner.PlayerActor.Trait<PlayerExperience>().GivePlayerExperience(exp / 100);
```

Comment:
Reuse the already declared Experience? (Has a default value of -1, but the same idea applies)